charts/internal/seed-controlplane/charts/vsphere-cloud-controller-manager/templates/vpa.yaml

```diff
@@ -12,7 +12,11 @@ spec:
     updateMode: Auto
   resourcePolicy:
     containerPolicies:
-    - containerName: '*'
+    - containerName: vsphere-cloud-controller-manager
       minAllowed:
         cpu: 20m
         memory: 40M
+      maxAllowed:
+        cpu: 4
+        memory: 10G
+      controlledValues: RequestsOnly
```

Comment:
I don't think we should be switching on `controlledValues: RequestsOnly` here. We didn't adjust the Limits to the values we've actually seen in production, so telling VPA to not scale Limits is pretty dangerous here.